examples/python/provider/__main__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016-2018, Pulumi Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from os import path
+from pulumi import ResourceOptions
+from pulumi_kubernetes import Provider
+from pulumi_kubernetes.core.v1 import Pod
+
+kubeconfig_file = path.join(path.expanduser("~"), ".kube", "config")
+with open(kubeconfig_file) as f:
+    kubeconfig = f.read()
```

Comment:
Ah, we should be careful here, as these are not the credentials for minikube, but our actual dev cluster. Just to be 100% explicit: can we think of _any possible way_ this will show up in the test logs? Based on a quick ctrl-f, it seems like the answer is no. Would just be nice to make sure we think hard about situations this might not be true, as this is likely not something we'd notice until we were compromised. :)

Reply:
I can't think of any way. We're already doing something like this in another example:

https://github.com/pulumi/pulumi-kubernetes/blob/69d6a815968ab99fa6cfb409176df1fad031e793/examples/provider/index.ts#L10

If we're confident that we won't print out this variable (which we aren't in either example), I think we should be fine.

Reply:
That said, this value will show up in the "diff view" of `pulumi up`. We don't do that, but if we did, we'd be printing it.